app/src/components/LayoutHeader.vue

```diff
@@ -42,30 +42,138 @@
         >
           {{ link.name }}
         </router-link>
+        <div class="border-b border-grey-400 my-4"></div>
+        <button @click="$emit('toggleAbout')" class="font-medium text-gray-500 hover:text-gray-900 uppercase">
+          About
+        </button>
+      </div>
+    </div>
+
+    <div v-if="userDisplayName" class="ml-auto flex items-center gap-x-4 md:gap-x-8">
+      <div class="flex items-center gap-x-2 h-14 group cursor-pointer">
+        <div class="hidden md:block group-hover:text-grey-500">Cart</div>
+        <CartIcon class="icon-small icon-primary" />
+        <div class="group-hover:text-grey-500">12</div>
+      </div>
+
+      <div class="border-r border-grey-100 h-14"></div>
+
+      <div class="group relative">
+        <div class="flex items-center h-16 gap-x-2 space-x-4 group cursor-pointer">
+          <div class="hidden md:block group-hover:text-grey-500">
+            <span>{{ userDisplayName }}</span>
+          </div>
+          <figure>
+            <Jazzicon :address="userAddress" :key="userAddress" width="40" />
+          </figure>
+          <div>
+            <ArrowBottomIcon class="icon-small icon-primary" />
+          </div>
+        </div>
+        <!-- menu-->
+        <div
+          class="
+            absolute
+            hidden
+            group-hover:flex
+            z-10
+            border border-grey-400
+            p-5
+            right-2
+            md:right-20
+            bg-white
+            text-grey-400
+            flex-col
+            gap-y-2
+            uppercase
+            font-medium
+            text-left
+            whitespace-nowrap
+          "
+        >
+          <div>{{ userDisplayName }}</div>
+          <div>{{ balance }} ETH</div>
```

Comment:
@apbendi @phutchins Why do we want to show a user's ETH balance in the header, given that 75% of user's check out with DAI? I think this should be removed as I'm not sure what value it adds